background/app/main_test.py

```diff
@@ -11,3 +11,100 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import os
+import pytest
+import uuid
+
+from google.cloud import firestore
+from google.cloud import pubsub
+
+os.environ['GOOGLE_CLOUD_PROJECT'] = os.environ['FIRESTORE_PROJECT_ID']
+os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = os.path.realpath(
+    os.path.join(
+        os.path.dirname(__file__),
+        '..',
+        '..',
+        'firestore-service-account.json'
```

Comment:
Can we add the service account from the standard `service-account.json` to the `FIRESTORE_PROJECT_ID` IAM? Then we wouldn't need a separate Firestore service account at all.

If not, can we create a separate `FIRESTORE_GOOGLE_APPLICATION_CREDENTIALS` environment variable?

I'm concerned about hardcoding `../../firestore-service-account.json`.

Reply:
After offline discuss, this sounds good. Can clean up/update at a later date.